GCRCatalogs/dc2_dm_catalog.py

```diff
@@ -299,14 +255,17 @@ def _obtain_native_data_dict(native_quantities_needed, native_quantity_getter):
         Overloading this so that we can query the database backend
         for multiple columns at once
         """
-        return native_quantity_getter.read_columns(list(native_quantities_needed), as_dict=True)
+        return native_quantity_getter.read_columns_row_group(list(native_quantities_needed),
+                                                             as_dict=True)
```

Comment:
Should we keep using `read_columns` here? I am not sure if any of existing catalogs have more than one row group.  If there is then this change will alter the behavior?